crates/hyperswitch_connectors/src/utils.rs

```diff
@@ -2740,391 +2810,390 @@ impl ForeignTryFrom<String> for NorwayStatesAbbreviation {
-                    "canton of jura" => Ok(Self::CantonOfJura),
-                    "canton of lucerne" => Ok(Self::CantonOfLucerne),
-                    "canton of neuchatel" => Ok(Self::CantonOfNeuchatel),
-                    "canton of schaffhausen" => Ok(Self::CantonOfSchaffhausen),
-                    "canton of solothurn" => Ok(Self::CantonOfSolothurn),
-                    "canton of st gallen" => Ok(Self::CantonOfStGallen),
-                    "canton of valais" => Ok(Self::CantonOfValais),
-                    "canton of vaud" => Ok(Self::CantonOfVaud),
-                    "canton of zug" => Ok(Self::CantonOfZug),
-                    "glarus" => Ok(Self::Glarus),
-                    "graubunden" => Ok(Self::Graubunden),
-                    "nidwalden" => Ok(Self::Nidwalden),
-                    "obwalden" => Ok(Self::Obwalden),
-                    "schwyz" => Ok(Self::Schwyz),
-                    "thurgau" => Ok(Self::Thurgau),
-                    "ticino" => Ok(Self::Ticino),
-                    "uri" => Ok(Self::Uri),
-                    "canton of bern" => Ok(Self::CantonOfBern),
-                    "canton of zurich" => Ok(Self::CantonOfZurich),
-                    _ => Err(errors::ConnectorError::InvalidDataFormat {
-                        field_name: "address.state",
-                    }
-                    .into()),
+            Err(_) => match value.as_str() {
+                "Aargau" => Ok(Self::Aargau),
+                "Appenzell Ausserrhoden" => Ok(Self::AppenzellAusserrhoden),
+                "Appenzell Innerrhoden" => Ok(Self::AppenzellInnerrhoden),
+                "Basel-Landschaft" => Ok(Self::BaselLandschaft),
+                "Canton of Fribourg" => Ok(Self::CantonOfFribourg),
+                "Canton of Geneva" => Ok(Self::CantonOfGeneva),
+                "Canton of Jura" => Ok(Self::CantonOfJura),
+                "Canton of Lucerne" => Ok(Self::CantonOfLucerne),
+                "Canton of Neuchâtel" => Ok(Self::CantonOfNeuchatel),
+                "Canton of Schaffhausen" => Ok(Self::CantonOfSchaffhausen),
+                "Canton of Solothurn" => Ok(Self::CantonOfSolothurn),
+                "Canton of St. Gallen" => Ok(Self::CantonOfStGallen),
+                "Canton of Valais" => Ok(Self::CantonOfValais),
+                "Canton of Vaud" => Ok(Self::CantonOfVaud),
+                "Canton of Zug" => Ok(Self::CantonOfZug),
+                "Glarus" => Ok(Self::Glarus),
+                "Graubünden" => Ok(Self::Graubunden),
+                "Nidwalden" => Ok(Self::Nidwalden),
+                "Obwalden" => Ok(Self::Obwalden),
+                "Schwyz" => Ok(Self::Schwyz),
+                "Thurgau" => Ok(Self::Thurgau),
+                "Ticino" => Ok(Self::Ticino),
+                "Uri" => Ok(Self::Uri),
+                "canton of Bern" => Ok(Self::CantonOfBern),
+                "canton of Zürich" => Ok(Self::CantonOfZurich),
```

Comment:
```suggestion
                "Canton of Bern" => Ok(Self::CantonOfBern),
                "Canton of Zürich" => Ok(Self::CantonOfZurich),
```
Should it be like this?

Reply:
yes it should start with lowercase (ref: https://github.com/juspay/hyperswitch-web/blob/main/src/States.json#L5679)

Reply:
How was this data created? ref: https://github.com/juspay/hyperswitch-web/blob/main/src/States.json#L5679
Can we trust it?